src/main/java/com/Acrobot/ChestShop/Listeners/ItemInfoListener.java

```diff
@@ -30,93 +30,108 @@
         }
-
+        
         if (meta instanceof EnchantmentStorageMeta) {
             for (Map.Entry<Enchantment, Integer> enchantment : ((EnchantmentStorageMeta) meta).getStoredEnchants().entrySet()) {
                 sender.sendMessage(ChatColor.YELLOW + capitalizeFirstLetter(enchantment.getKey().getName(), '_') + ' ' + toRoman(enchantment.getValue()));
             }
         }
     }
-
-    @EventHandler
-    public static void addPotionInfo(ItemInfoEvent event) {
+    
+    @EventHandler(ignoreCancelled = true)
+    public static void addPotionInfo (ItemInfoEvent event) {
         ItemStack item = event.getItem();
-
+        
         if (item.getType() != Material.POTION || item.getDurability() == 0) {
             return;
         }
-
+        
         Potion potion;
-
+        
         try {
             potion = Potion.fromItemStack(item);
         } catch (IllegalArgumentException ex) {
             return;
         }
-
+        
         StringBuilder message = new StringBuilder(50);
-
+        
         message.append(ChatColor.GRAY);
-
+        
         if (potion.getType() == null) {
             return;
         }
-
+        
         if (potion.isSplash()) {
             message.append("Splash ");
         }
-
+        
         message.append("Potion of ");
         message.append(capitalizeFirstLetter(potion.getType().name(), '_')).append(' ');
         message.append(toRoman(potion.getLevel()));
-
+        
         CommandSender sender = event.getSender();
-
+        
         sender.sendMessage(message.toString());
-
+        
         for (PotionEffect effect : potion.getEffects()) {
             sender.sendMessage(ChatColor.DARK_GRAY + capitalizeFirstLetter(effect.getType().getName(), '_') + ' ' + toTime(effect.getDuration() / 20));
         }
     }
-
-    @EventHandler
-    public static void addBookInfo(ItemInfoEvent event) {
+    
+    @EventHandler(ignoreCancelled = true)
+    public static void addBookInfo (ItemInfoEvent event) {
         ItemMeta meta = event.getItem().getItemMeta();
         if (meta instanceof BookMeta) {
             BookMeta book = (BookMeta) meta;
-            iteminfo_book.send(event.getSender(),
-                    "title", book.getTitle(),
-                    "author", book.getAuthor(),
-                    "pages", String.valueOf(book.getPageCount())
-            );
+            iteminfo_book.send(event.getSender(), "title", book.getTitle(), "author", book.getAuthor(), "pages", String.valueOf(book.getPageCount()));
             if (book.hasGeneration()) {
-                iteminfo_book_generation.send(event.getSender(),
-                        "generation", StringUtil.capitalizeFirstLetter(book.getGeneration().name(), '_')
-                );
+                iteminfo_book_generation.send(event.getSender(), "generation", StringUtil.capitalizeFirstLetter(book.getGeneration().name(), '_'));
```

Comment:
These seem to be unnecessary code changes too which only restructure how the code is written and not actually change the function/aren't related to the PRs change at all? Should be reverted to keep diff small.